src/Server/middleware/userRequiredMiddleware.ts

```diff
@@ -9,6 +9,7 @@ const USER_REQUIRED_ROUTES = [
   "/settings/purchases(.*)",
   "/settings/shipping",
   "/user/conversations(.*)",
+  "/collector-profile/saves2(.*)",
```

Comment:
**Note:** We show a My Collection landing page whenever you land on any URL that starts with `/collector-profile/` when you are logged out, and after login we forward the user to the previously requested link.

This way works as well, but I just wanted to share the current implementation :)